website/docs/d/remote_location.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mso"
+page_title: "MSO: mso_remote_location"
+sidebar_current: "docs-mso-resource-remote-location"
+description: |-
+  Data source for MSO Remote Location
+---
+
+# mso_user #
+
+Data source for MSO Remote Location
+
+## Example Usage ##
+
+```hcl
+
+data "mso_remote_location" "password" {
+  name = "remote_location_password"
```

Comment:
name = "remote_location_password" -> "remote_location"

Reply:
This was a little confusing to read at first

Reply:
what do you mean? I kept different names for the resource on purpose, what name would you propose?

Reply:
```
data "mso_remote_location" "password" {
    name = "remote_location_name"
}
```